### ISSUE TITLE
Sketch: add an option to disable snapping and/or limit snap thresold

### ISSUE THREAD

**Issue by jacobwod** [hajkmap/Hajk#1386]
On some occasions the snap functionality is rather an obstacle than a helping function. 

In the screenshot below user would like to align the two features next to each other, but not exactly on top of each other. Right now it's impossible, as the dragged feature will automatically snap onto the other one as soon as we get within a certain pixel threshold:
![Skärmavbild 2023-08-28 kl  15 05 11](https://github.com/hajkmap/Hajk/assets/110222/f89f0936-1811-426e-8cbc-36d4d50320ed)

Proposal A: Add an option that controls the snapping threshold in Sketch
Proposal B: Add an option to disable/enable snapping in Sketch

The proposals are not mutually exclusive.

**Comment by jesade-vbg** [comment]
I agree, the threshold should be possible to change. Measurer has the same issues as they both use Drawmodel and uses the same threshold value.

**Comment by jesade-vbg** [comment]
This is handled in https://github.com/hajkmap/Hajk/issues/1616, which includes a solution for it.

**Comment by jesade-vbg** [comment]
Can this be closed? The ability to disable snap was added here https://github.com/hajkmap/Hajk/issues/1616

Or keep it because you can't finetune as in proposal A?
